### ISSUE TITLE
NVDA in Firefox does not announce that an item is expanded and the focused item from the submenu

### ISSUE THREAD

**Issue by nelito987** [telerik/kendo-ui-core#6253]
### Bug report

Currently, when an item is expanded in the Menu, NVDA in Firefox does not read which item/subitem is expanded. Apart from that, the reader does not announce the newly focused item from the submenu.

https://www.w3.org/TR/wai-aria-practices-1.2/examples/menubar/menubar-navigation.html

As a comparison, NVDA in Chrome, ChromeVox in Chrome, and JAWS in IE announce the same in both the Kendo Menu and the above WAI-ARIA practices example.

**_Edit_**: Same problem occurs in Chrome, check https://github.com/telerik/kendo-ui-core/issues/6253#issuecomment-1425403711

### Reproduction of the problem
1. Open the [demos](https://demos.telerik.com/kendo-ui/menu/keyboard-navigation)
2. Focus on the Menu and expand an item.

### Current behavior
The expanded state is not announced by NVDA

### Expected/desired behavior
The screen reader should be able to read that an item is expanded in the Menu.

### Environment
* **Kendo UI version:** 2021.1.119
* **Browser:** [all  ]

**Comment by veselints** [comment]
There are two important differences between the Kendo Menu implementation (rendering) and the example from the WAI-ARIA practices:
- The WAI-ARIA example uses focusable `menuitems`, while the Kendo Menu keeps the focus on the `menubar` element and uses `aria-activedescendant`;
- The `menuitems` in the Kendo Menu hold their owned `submenus` while the WAI-ARIA example have the `menuitem` and its `submenu` as siblings;

Both the above could cause the issue in question. Using the `aria-activedescendant`, however, is among the recommended approaches by WAI-ARIA:
https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties-13

**Comment by veselints** [comment]
Moving the `menuitem` role to the link element (along with all the ARIA attributes) and use `role="none"` on the list elements does not resolve the issue in FF and NVDA.

**Comment by gdenchevprog** [comment]
The problem with NVDA and FireFox seems to be resolved, however the issue now appears in Chrome(both with NVDA and JAWS).

To test this, open the following demo in Chrome and run a screen reader:
https://demos.telerik.com/kendo-ui/menu/remote-data-binding

When you focus a menu element, expand it, and then you focus a subitem the reader doesn't announce in anything. Compare that to FireFox where it reads out the focused subitem.

Also raising priority as this has been reported by a client in ticket 1596926.

**Comment by nelito987** [comment]
Reported for ContextMenu (in Chrome) in ticket: 1607309

**Comment by gdenchevprog** [comment]
Raising again - 1621525

**Comment by nelito987** [comment]
Reported in ticket ID: 1631988
Note: NVDA reads correctly in Chrome the items and subitems in the Kendo Angular Menu - https://www.telerik.com/kendo-angular-ui/components/menus/menu

**Comment by pepinho24** [comment]
> Reported in ticket ID: 1631988 Note: NVDA reads correctly in Chrome the items and subitems in the Kendo Angular Menu - https://www.telerik.com/kendo-angular-ui/components/menus/menu

All the other flavors use a different focusing mechanism - they focus the item element itself, while in jQuery the actual focus remains on the main element.
